packages/drivers/odsp-driver/src/odspSnapshotParser.ts

```diff
@@ -8,6 +8,19 @@ import * as api from "@fluidframework/protocol-definitions";
 import { IOdspSnapshot, IOdspSnapshotCommit } from "./contracts";
 import { ISnapshotContents } from "./odspUtils";
 
+/** https://portal.microsofticm.com/imp/v3/incidents/details/308931013/home
+ * The commits property was removed from protocol-definitions but in order to support back compat, we will
+ * temporarily add back in this local structure in order to upload the snapshot to support rolling back to 0.58.
+ * Notice this entire interface will be removed once the backward compatibility is not required anymore.
+*/
+ interface ISnapshotTree {
```

Comment:
I think you can just extend the original interface instead of copying the whole interface.